challenge_set_a_table/src/challenge_set_a_table_states/manipulate_machine.py

```diff
@@ -132,7 +137,12 @@ def __init__(self, robot, place_designator):
     def execute(self, userdata=None):
         # Try to place the object
         item = ds.EdEntityDesignator(robot=self._robot, id=arm.occupied_by.id)
-        arm_designator = ds.OccupiedArmDesignator(self._robot)
+        arm_designator = ds.OccupiedArmDesignator(self._robot, arm_properties={"required_trajectories": ["prepare_place"
+                                                                                                         ],
+                                                                               "required_goals": ["reset",
+                                                                                                  "handover_to_human"],
```

Comment:
Maybe move the `arm_properties=` to the next line? that would give you a little extra space to cleanup the layout.